packages/terra-consumer-nav/src/components/modal/Modal.jsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
this is definitely feasible and makes sense to me. I can do that separately from this PR if that's better.